src/main/java/digital/slovensko/autogram/ui/gui/SignatureLevelStringConverter.java

```diff
@@ -0,0 +1,30 @@
+package digital.slovensko.autogram.ui.gui;
+
+import eu.europa.esig.dss.enumerations.SignatureLevel;
+import javafx.util.StringConverter;
+
+
+public class SignatureLevelStringConverter extends StringConverter<SignatureLevel> {
```

Comment:
Túto classu by som hodil niekam do Util. Možno aj priamo tie dve metódy hodiť do DSSUtils.java. A všetky metódy by som spravil static, nech netreba vyrábať zbytočne inštanciu tejto classy.

Reply:
Alebo do GUIUtils

Reply:
Oh, I see. To sa používa v tých dropdownoch. Tak asi nič potom :/